modules/dns_nsupdate/dns_nsupdate_gss_plugin.rb

```diff
@@ -0,0 +1,17 @@
+module ::Proxy::Dns::NsupdateGSS
+  class Plugin < ::Proxy::Provider
+    plugin :dns_nsupdate_gss, ::Proxy::VERSION,
+           :factory => proc { |attrs| ::Proxy::Dns::NsupdateGSS::Record.record(attrs) }
+
+    default_settings :dns_server => 'localhost', :dns_ttl => 86_400,
```

Comment:
Sorry, the *dns_ttl* default specifically, since it's in the main dns.yml config.